modules/traf/files/buildtest/stage-traf.sh

```diff
@@ -49,9 +49,9 @@ BLD="$(< $workspace/Build_ID)"
 
 if [[ "$Flavor" == "debug" ]]
 then
-  FileSuffix="_debug-$BLD.tar.gz"
+  FileSuffix="_debug-$BLD-incubating-bin.tar.gz"
```

Comment:
Do we want to be consistent in usage of "-" and "_"?  I see _debug, should it be -debug instead?